packages/edit-post/src/components/block-settings-menu/test/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies.
+ */
+import ShallowRenderer from 'react-test-renderer/shallow';
+
+/**
+ * Internal dependencies.
+ */
+import { PluginBlockSettingsMenuGroupSlot } from '../plugin-block-settings-menu-group';
+
+describe( 'PluginBlockSettingsMenuGroupSlot', () => {
+	test( 'should render normally if block selected has name property', () => {
+		const renderer = new ShallowRenderer();
+		renderer.render(
+			<PluginBlockSettingsMenuGroupSlot selectedBlocks={ [ { name: 'paragraph' } ] } />
+		);
+
+		expect( renderer.getRenderOutput() ).toMatchSnapshot();
+	} );
+
+	test( 'should be hidden if no block is selected', () => {
+		const renderer = new ShallowRenderer();
+		renderer.render(
+			<PluginBlockSettingsMenuGroupSlot selectedBlocks={ [ null ] } />
```

Comment:
The question here is why would it ever happen in the first place and? It looks buggy. There should be always a valid object passed here.